app/policies/plan_policy.rb

```diff
@@ -56,6 +56,10 @@ def visibility?
     @plan.administerable_by?(@user.id)
   end
 
+  def set_test?
+    @plan.administerable_by?(@user.id)
```

Comment:
they both agreed that owners and co-owners should be able to change visibility and set the test flag